cpp/tests/io/csv_test.cpp

```diff
@@ -492,39 +500,29 @@ TEST_F(CsvReaderTest, MultiColumn)
   cudf_io::csv_reader_options in_opts =
     cudf_io::csv_reader_options::builder(cudf_io::source_info{filepath})
       .header(-1)
-      .dtypes(std::vector<std::string>{"int8",
-                                       "short",
-                                       "int16",
-                                       "int",
-                                       "int32",
-                                       "long",
```

Comment:
Don't we need to support long and other type names ?

Reply:
They are and will stay supported in the Python API. We do support other types here (`short`, `int`, `long`, etc.), but there are platform-specific, so not encouraged IMO.
I can keep `long`, `short` types in the test, but IMO they don't add to the test, and are not guaranteed to work correctly on all platforms.